fcrepo-kernel-modeshape/src/main/java/org/fcrepo/kernel/modeshape/rdf/impl/BlankNodeRdfContext.java

```diff
@@ -1,122 +0,0 @@
-/**
- * Copyright 2015 DuraSpace, Inc.
- *
- * Licensed under the Apache License, Version 2.0 (the "License");
- * you may not use this file except in compliance with the License.
- * You may obtain a copy of the License at
- *
- *     http://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing, software
- * distributed under the License is distributed on an "AS IS" BASIS,
- * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- * See the License for the specific language governing permissions and
- * limitations under the License.
- */
-package org.fcrepo.kernel.modeshape.rdf.impl;
-
-import com.google.common.base.Function;
-import com.google.common.base.Predicate;
-import com.google.common.collect.ImmutableList;
-import com.google.common.collect.Iterators;
-import com.hp.hpl.jena.rdf.model.Resource;
-
-import org.fcrepo.kernel.api.models.FedoraResource;
-import org.fcrepo.kernel.api.exception.RepositoryRuntimeException;
-import org.fcrepo.kernel.api.identifiers.IdentifierConverter;
-import org.fcrepo.kernel.api.utils.iterators.RdfStream;
-import org.fcrepo.kernel.modeshape.rdf.impl.mappings.PropertyValueIterator;
-
-import javax.jcr.Node;
-import javax.jcr.Property;
-import javax.jcr.RepositoryException;
-import javax.jcr.Value;
-
-import java.util.Iterator;
-import java.util.List;
-
-import static java.util.Arrays.asList;
-import static javax.jcr.PropertyType.PATH;
-import static javax.jcr.PropertyType.REFERENCE;
-import static javax.jcr.PropertyType.WEAKREFERENCE;
-import static org.fcrepo.kernel.modeshape.identifiers.NodeResourceConverter.nodeConverter;
-import static org.fcrepo.kernel.modeshape.utils.FedoraTypesUtils.isBlankNode;
-
-/**
- * Embed all blank nodes in the RDF stream
- *
- * @author cabeer
- * @author ajs6f
- * @since 10/9/14
- */
-public class BlankNodeRdfContext extends NodeRdfContext {
```

Comment:
I have encountered similar issues at other times, when a file is renamed _and_ changed substantially. The issue is this: git tracks content, not files.